src/components/auth/signIn/email/SignInModalContent.module.sass

```diff
@@ -0,0 +1,101 @@
+@import 'src/styles/subsocial-vars.scss'
+
+.SignInModalContent
+    display: flex
+    flex-direction: column
+    justify-content: center
+    padding: 0 $space_tiny
+    gap: $space_normal
+    margin-top: $space_big
+
+.ButtonLinkDiv
+    font-size: $font_normal
+    font-weight: $font_weight_semibold
+    margin-left: $space_mini
+    padding: 0px
+
+.ButtonLinkMedium
+    font-size: $font_normal
+    font-weight: $font_weight_medium
+    &:disabled
+        color: #888888
+
+    &:disabled:hover
+        color: #888888
+
+.ConfirmationStepContent
+    @extend .SignInModalContent
+    align-items: center
+    gap: $space_big
+
+
+.InnerCard
+    border-radius: 8px
+    border: 1px solid #d9d9d9
+    :global(.ant-card-body)
+        display: flex
+        justify-content: space-between
+        align-items: center
+        padding: $space_semilarge $space_small
+
+.InnerDivider
+    height: 3rem
+    border-left: 1px solid #dddddd
+
+.MnemonicText
+    text-align: left
+    font-weight: 400
+    font-size: $font_normal
+
+.CopyIcon
+    padding: 0 6.5px
+    color: #888888
+    &:hover
+        color: $color_link_hover
+
+.BaseFormItem
+    margin-bottom: 0
+    :global(.ant-form-item-explain)
+        font-weight: $font_weight_normal
+        font-size: $font_small
+        line-height: 19px
+        color: $color_error
+
+.BaseFormItemError
+    @extend .BaseFormItem
+    border-color: $color_error
+
+.CodeFormItem
+    @extend .BaseFormItem
+    :global(.ant-form-item-explain)
+        text-align: center
+        margin-right: $space_semismall
+        margin-top: $space_tiny
+
+
+.InputOTP
+  width: 40px
+  height: 44px
+  padding: 0
+
+  font-size: $font_big
+  text-align: center
+  font-weight: $font_weight_medium
+  line-height: 140%
+  color: #222222
```

Comment:
use var

Reply:
Applied in [21ffc94](https://github.com/dappforce/polkaverse/pull/74/commits/21ffc947098855e4f1a52b34e2f240282e4ee4da)